modules/remind.py

```diff
@@ -109,10 +79,48 @@ def monitor(jenni):
     'sec': 1,
     's': 1
 }
+r_command = None
+
+
+def setup(jenni):
+    global r_command
+
+    periods = '|'.join(scaling.keys())
+    p_command = r'{}in ([0-9]+(?:\.[0-9]+)?)\s?((?:{})\b)?:?\s?(.*)'.format(
+        jenni.config.prefix,
+        periods,
```

Comment:
I do that so that if in the future if these an item is added or removed from this function call git diff will show only the lines added or removed instead of n+1 lines for fixing the additional comma on the ending line.